services/funfuzz/recipes/install_prerequisites.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -ex
+
+echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
```

Comment:
This is already done here: https://github.com/MozillaSecurity/orion/blob/master/base/fuzzos/recipes/fuzzos.sh#L9 and shouldn't be necessary.